app/components/agenda/agendaitem/agendaitem-controls.hbs

```diff
@@ -1,23 +1,32 @@
 {{#if @currentAgenda.status.isDesignAgenda}}
   {{#if
-    (and (not (is-pending this.isDeletable)) (not (is-pending this.isPostPonable)))
+    (or this.isDeletable this.areDecisionActionsEnabled)
   }}
     <WebComponents::AuButtonToolbar>
       <Auk::DropdownMenu
         @label={{t "actions"}}
         class="auk-u-hidden@print"
         data-test-agendaitem-controls-actions={{true}}
       as |Menu|>
-        {{#if this.isPostPonable}}
-          {{#if @agendaitem.retracted}}
-            <Menu.Item {{on "click" (fn this.advanceAgendaitem @agendaitem)}} data-test-agendaitem-controls-action-advance>
-              {{t "not-postpone"}}
-            </Menu.Item>
-          {{else}}
-            <Menu.Item {{on "click" (fn this.postponeAgendaitem @agendaitem)}} data-test-agendaitem-controls-action-postpone>
-              {{t "postpone"}}
-            </Menu.Item>
-          {{/if}}
+        {{#if this.areDecisionActionsEnabled}}
+          {{#let this.decisionActivity.decisionResultCode as |drc| }}
```

Comment:
Not a fan of abbreviated variable names as they make the code rather cryptic. Maybe use `resultCode` or just `code`?

Reply:
Can change, no problem.
As to the approach of using `#let`, is that ok?
I could have just used the long path twice and then there would be no need to have an alias.

Reply:
`resultCode` sounds less vague down the line. using that

Reply:
No strong opinion about the `{{#let}}`. I get that it might sometimes be useful to make things more readable in templates.